pkg/controller/controller_broker_test.go

```diff
@@ -89,13 +89,24 @@ func TestShouldReconcileBroker(t *testing.T) {
 		{
 			name: "ready, interval not elapsed",
 			broker: func() *v1alpha1.Broker {
-				broker := getTestBrokerWithStatus(v1alpha1.ConditionTrue)
+				broker := getTestBrokerWithStatus(v1alpha1.ConditionTrue, nil)
 				return broker
 			}(),
 			now:       time.Now(),
 			interval:  3 * time.Hour,
 			reconcile: false,
 		},
+		{
+			name: "ready, interval not elapsed, checksum changed",
+			broker: func() *v1alpha1.Broker {
```

Comment:
Nit: you can leave the getTestBrokerWithStatus func unchanged and set the checksum on the status in this anonymous func.  I generally keep single-cases of specialization in anon funcs like this one instead of modifying a widely used `getTestThing` func.